maven-plugin-api/src/main/mdo/plugin.mdo

```diff
@@ -24,9 +24,10 @@ under the License.
   <description><![CDATA[
     Plugin descriptor, stored in <code>META-INF/maven/plugin.xml</code> in a plugin's jar artifact.
     This descriptor is generally generated from plugin sources, using
-    <a href="/plugins/maven-plugin-plugin/">maven-plugin-plugin</a>.
-    <p><i>Notice:</i> this documentation is generated from a Modello model but the code executed is not generated
-    from this descriptor. Please report if you find anything wrong.</p>
+    <a href="/plugins/maven-plugin-plugin/">maven-plugin-plugin</a>, but other tools may geenrated it or it could even be hand-written.
```

Comment:
I'm not really about adding the sentence that `other tools may generated it or it could even be hand-written`.  Maybe I'm wrong, but I'm not aware of any such tools.  If there's none, this becomes irrelevant.
Also from an API point of view, I would consider the xml plugin descriptor an internal technical feature, rather than a user facing API, as really, users don't manipulate or see it.  It just happens to be generated by the tooling provided by maven and understood by maven itself.

Reply:
It happens the file is written manually and it is regularly parsed by other tools (rarely generated I guess, it is either the official generator or manually done).
But this is a public API since it is part of the contract of plugins, the tools don't encapsulate anything while it must be published IMHO so structure has to be stable even if it can evolve (additions and things like that are ok for ex for such cases).

Reply:
ok, "is generally generated from plugin sources using maven-plugin-plugin" is sufficient, no absolute need to add more: I'll remove the extra explanation

Reply:
The two cases I met and think are ok were:

1. Don't depend on the maven build stack for a simple plugin (= very few primitives as param) - this one was "old"
2. No maven in the build pipeline (saw plain makefile like to build "build tools" and some sbt case)

That said no issue to drop the extra part, just wanted to highlight we might not want to enforce the usage of a maven plugin and keep it as a contract since it is what it is by design.